apps/rush-lib/src/cli/actions/InitAutoinstaller.ts

```diff
@@ -0,0 +1,63 @@
+import * as colors from 'colors';
+
+import { CommandLineStringParameter } from '@rushstack/ts-command-line';
+import { FileSystem, NewlineKind, IPackageJson, JsonFile } from '@rushstack/node-core-library';
+
+import { BaseRushAction } from './BaseRushAction';
+import { RushCommandLineParser } from '../RushCommandLineParser';
+import { Autoinstaller } from '../../logic/Autoinstaller';
+
+export class InitAutoinstallerAction extends BaseRushAction {
+  private _name: CommandLineStringParameter;
+
+  public constructor(parser: RushCommandLineParser) {
+    super({
+      actionName: 'init-autoinstaller',
+      summary: 'Initializes a new autoinstaller',
+      documentation:
+        'Use this command to initialize a new autoinstaller folder.  Autoinstallers provide a way to' +
+        ' manage a set of related dependencies that are used for scripting scenarios outside of the usual' +
+        ' "rush install" context.  See the command-line.json documentation for an example.',
+      parser
+    });
+  }
+
+  protected onDefineParameters(): void {
+    this._name = this.defineStringParameter({
+      parameterLongName: '--name',
+      argumentName: 'AUTOINSTALLER_NAME',
+      required: true,
+      description:
+        'Specifies the name of the autoinstaller folder, which must conform to the naming rules for NPM packages.'
+    });
+  }
+
+  protected async run(): Promise<void> {
+    const autoinstallerName: string = this._name.value!;
+
+    const autoinstaller: Autoinstaller = new Autoinstaller(autoinstallerName, this.rushConfiguration);
+
+    if (FileSystem.exists(autoinstaller.folderFullPath)) {
+      // It's okay if the folder is empty
+      if (FileSystem.readFolder(autoinstaller.folderFullPath).length > 0) {
```

Comment:
Why not just read this folder and ignore thrown not-exist errors.

Reply:
I prefer to avoid unexceptional exceptions.